docs/manuals/gui/configuration.md

```diff
@@ -110,6 +110,8 @@ Here is the list of the configuration parameters you can use in
      application to the Internet. See the section on
      [Accessing your app from the Web](#accessing-your-app-from-the-web) for details.
 
+!!! info "For running Taipy Gui service with some other Taipy services, please refer to page [Running Taipy services](../running_services/index.md)"
```

Comment:
"For running Taipy Gui service"
->
"To run the Taipy GUI service"

"to page [Running Taipy services](../running_services/index.md)"
to
"to the [Running Taipy services](../running_services/index.md) page"